### ISSUE TITLE
Visual design: left nav

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3083]
- Expand/collapse control on the left
- Dot on the right indicates whether content has been entered for that field
- Expanded properties and all properties nested under it are black. If user clicks 'Save' and required fields are not entered, they will display in red to indicate they need to be filled in.
- Collapsed properties are in blue.
- Is there any opportunity to save what the user has expanded/collapsed so the next time the resource is opened, the same properties are expanded/collapsed?
![image](https://user-images.githubusercontent.com/42216982/135568403-8fd00d7a-e2c1-46e0-a42b-d7a62fc8cf51.png)

**Comment by jcoyne** [comment]
@astridu This is a bit too abstract for me to implement. Can you demonstrate this with real properties that we can see in stage?

**Comment by astridu** [comment]
Does this help?

![image](https://user-images.githubusercontent.com/42216982/135941992-873793a7-f12e-4a7f-b3a6-1391b642e34e.png)

**Comment by jcoyne** [comment]
Yes, this is much easier to understand. Thanks for doing the extra work @astridu

**Comment by jcoyne** [comment]
Developers; It looks like this removes the `btn-primary` class from the sidebar.  When this happens, also remove it from `resourceNav.scss` and remove `$vanilla` from `colors.scss`

**Comment by jcoyne** [comment]
@astridu I'm looking for a mockup that mixes in the expand controls with those that don't get expand controls.   How should those be positioned when we have a mix of hierarchical and non-hierarchical
